src/chaos-experiments/models/database/databaseConnector.js

```diff
@@ -57,16 +57,16 @@ async function insertChaosJobExperiment(jobExperimentId, jobId, experimentId, st
     return databaseConnector.insertChaosJobExperiment(jobExperimentId, jobId, experimentId, startTime, endTime, contextId);
 }
 
-async function getChaosJobExperimentById(jobExperimentId, contextId) {
-    return databaseConnector.getChaosJobExperimentById(jobExperimentId, contextId);
+async function getChaosJobExperimentsByJobId(jobExperimentId, contextId) {
+    return databaseConnector.getChaosJobExperimentsByJobId(jobExperimentId, contextId);
 }
 
 async function getChaosJobExperimentByJobId(jobId, contextId) {
     return databaseConnector.getChaosJobExperimentById(jobId, contextId);
```

Comment:
general comment not related to this MR, it seems weird that in a file called databaseConnector.js, we have databaseConnector which is a different entity (I guess its sequalizeConnector)

Reply:
seems like an issue in all the repository
there's the databaseConntor main and the sequalizeConnector that is called a connector

Reply:
cool but maybe not call it here database connector? maybe rename to connector?